testsuite/MDAnalysisTests/core/test_unwrap.py

```diff
@@ -37,13 +37,21 @@ class TestUnwrap(object):
     which is specifically designed for wrapping and unwrapping tests.
     """
     precision = 5
+    reference_point_pos = (None,
+                           np.array([5, 5, 5], dtype=np.float32),
```

Comment:
I would avoid having x, y, and z set to the same value. If you have an error in your indices it may not make the test fail.

Reply:
Sure.